scripts/radix_git_status.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+set completion-ignore-case on
+set show-all-if-ambiguous on
+
+ERROR_MSG_NO_SUCH_FOLDER="___folder_does_not_exist___"
+
+RADIXDLT_JAVA_COMMON_FIRST_COMMIT_HASH='1b6b6c46c9a5b87ec29a32b47bf96e15bdfbf165'
+
+RADIXDLT_JAVA_CLIENT_LIBRARY_NAME_OF_REPO_AND_ASSUMED_NAME_OF_DIRECTORY='radixdlt-java'
+RADIXDLT_CORE_NAME_OF_REPO_AND_ASSUMED_NAME_OF_DIRECTORY='radixdlt-core'
+RADIXDLT_ENGINE_NAME_OF_REPO_AND_ASSUMED_NAME_OF_DIRECTORY='radix-engine-library'
+# co-co-co-combo breaker!
+RADIXDLT_REGRESSION_TESTS_NAME_OF_REPO_AND_ASSUMED_NAME_OF_DIRECTORY='RadixRegressionTests' 
+
+# Arguments:
```

Comment:
Would be good if this script worked, even if it was not run with the current directory the same as the the script directory, eg `./scripts/radix_git_status.sh`

You could use `scriptdir=$(dirname "$0")` to work out where the script directory was relative to the working directory it was run from.

Reply:
nice! fixed! would be nice to get it working even when using symlinks, but `readlink -f` does not work on macOS 🤷‍♂️